CONTRIBUTING.md

```diff
@@ -1,16 +1,88 @@
-# Contributing guide
+# Contribution guide
 
 This contribution guide aims to help you with the optimal way to get involved with this project.
 
-Please contact us if you need help or have ideas to improve this guide.
+## Table of contents
 
-## Content contributions (market data)
+* [Market data contributions](#market-data-contributions)
+* [Code contributions](#code-contributions)
+* [How to use Git](#how-to-use-git)
+* [General advices](#general-advices)
+
+Please do not hesitate to contact us if you need help or have ideas on how to improve this guide.
+
+
+## Market data contributions
+
+You can add market data for your city in a few steps. All you need to do is to **add one file** (_city.json_)
+and to **modify two existing** ones (_cities.json_, _README.md_). Here is how you start:
+
+![city.json][icon-city-json] ![cities.json][icon-cities-json] ![README.md][icon-readme-md]
+
+1. Add the market data for your _city.json_ as a [GeoJSON][geojson-org] file.
+   You might wanna use **German spelling** with Umlauts for city names which is supported.
+2. Edit [*cities/cities.json*](cities/cities.json) to add your city to the list of cities.
+3. Edit the [*README.md*](README.md) to add your city to the list of **supported cities**.
+
+
+### Consistent formatting
+
+* It would be great if you could apply **automatic formatting** for the GeoJSON file using the editor of your choice.
+* Please stick to the formatting when editing existing files.
+
+
+### Market data format specification
+
+This section describes the file format which is used to store the market data for an individual city.
+
+* The market data is stored in [GeoJSON format][geojson-org].
+  Have a look at *[cities/karlsruhe.json](cities/karlsruhe.json)* to learn about the general structure of the file.
```

Comment:
Using `cities/karlsruhe.json` for the link leads to a graphical display of the data. I think the [text view](https://github.com/wo-ist-markt/wo-ist-markt.github.io/blob/master/cities/karlsruhe.json?short_path=e135580) would be better.